toolkit/tools/imageconfigvalidator/imageconfigvalidator.go

```diff
@@ -145,7 +145,7 @@ func validatePackages(config configuration.Config) (err error) {
 			if pkg == dracutFipsPkgName {
 				foundDracutFipsPackage = true
 			}
-			if pkg == selinuxPkgName {
+			if strings.Contains(pkg, selinuxPkgName) {
```

Comment:
If you want to do the custom config route, it would have to be part of `kernelcommandline.go`. `selinux.go` defines a simple type that is just a string, would have to add logic to select between a string, or a struct like
```golang
type SELinux struct {
    SELinuxMode SELinux `json:"SELinuxMode "`
    SELinuxPolicy string    `json:"SELinuxPolicy"`
}
```
inside the unmarshal code... getting a bit complex.

Reply:
I would just add a `SELinuxPolicyPackage` field to the `KernelCommandLine` struct.

The crazy parsing hacks I added to `AdditionalFiles` should be avoided unless there aren't any other options.

Reply:
Thanks guys for the suggestions!